src/ai/backend/common/message_queue/redis_queue.py

```diff
@@ -218,7 +219,9 @@ async def _failover_consumer(self, e: redis.exceptions.ResponseError) -> None:
         # and start the auto claim loop again
         if "NOGROUP" in str(e):
             log.warning(
-                f"Consumer group {self._group_name} does not exist, creating it",
```

Comment:
Is there a reason not to use f-strings?
I think f-string format looks more readable.

Reply:
By default, format string will always be formatted, but format in logs will not format If the loglevel is low.